src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/WindowsForms/MyAppFileAccessor.cs

```diff
@@ -176,5 +176,5 @@ private async Task SetPropertyAsync(string propertyName, string value)
 
     public async Task<int?> GetMinimumSplashScreenDisplayTimeAsync() => await GetIntPropertyValueAsync(MinimumSplashScreenDisplayTimeProperty);
 
-    public async Task SetMinimumSplashScreenDisplayTimeAsync(int value) => await SetPropertyAsync(MinimumSplashScreenDisplayTimeProperty, value.ToString());
+    public async Task SetMinimumSplashScreenDisplayTimeAsync(int? value) => await SetPropertyAsync(MinimumSplashScreenDisplayTimeProperty, value.ToString());
```

Comment:
This will throw if it receives null, as is. It should have given a compiler warning, I would have expected.

```suggestion
    public async Task SetMinimumSplashScreenDisplayTimeAsync(int? value) => await SetPropertyAsync(MinimumSplashScreenDisplayTimeProperty, value?.ToString());
```